ironmon_tracker/screens/LogOverlay.lua

```diff
@@ -33,7 +33,8 @@ LogOverlay = {
 	currentEvoSet = 1, -- Ideally move this somewhere else
 	prevEvosPerSet = 1,
 	evosPerSet = 3, -- Ideally move this somewhere else
-	preEvoSetting = "Show Pre Evolutions"
+    preEvoSetting = "Show Pre Evolutions",
```

Comment:
Looks like the spacing is using actual spaces here instead of tabs. Can you revert this back to using tabs like the line above and below it.